src/main/java/net/mcreator/ui/MCreator.java

```diff
@@ -258,6 +277,32 @@ else if (screenSize.getWidth() > 1290 && screenSize.getHeight() > 795)
 		add("Center", splitPane);
 	}
 
+	private static List<File> loadUserBackgrounds() {

```

Comment:
These two methods should be moved to some other place, maybe to init package into class called eg. BackgroundLoader